specification/digitaltwins/data-plane/Microsoft.DigitalTwins/preview/2023-02-27-preview/examples/BulkImportJobCancel.json

```diff
@@ -1,17 +1,16 @@
 {
```

Comment:
Should we change the names of the example files as well? e.g BulkImportJobCancel --> ImportJobCancel

Reply:
These names are not referred anywhere, just the content

Reply:
Seems like it could lend to mistakes in the future though - like keeping the Bulk name there when we don't want it in new files